src/server/models/tag.ts

```diff
@@ -0,0 +1,65 @@
+import Sequelize from 'sequelize'
+import { sequelize } from '../util/sequelize'
+import { IdType } from '../../types/server/models'
+import { TAG_KEY_REGEX, TAG_STRING_REGEX } from '../../shared/util/validation'
+
+export interface TagType extends IdType, Sequelize.Model {
+  readonly tagString: string
+  readonly tagKey: string
+}
+
+// For sequelize define
+type TagTypeStatic = typeof Sequelize.Model & {
+  new (values?: object, options?: Sequelize.BuildOptions): TagType
+}
+
+export const Tag = <TagTypeStatic>sequelize.define(
+  'tag',
+  {
+    tagString: {
+      type: Sequelize.STRING(25),
```

Comment:
@halfwhole raised a good point, wanted to suggest not constraining this here so that we don't have to perform future migrations if we want to support longer tag lengths

Reply:
🤔 okay, will change this to 255